src/PhpSpec/Listener/MethodNotFoundListener.php

```diff
@@ -63,6 +88,11 @@ public function afterSuite(SuiteEvent $event)
             return;
         }
 
+        if (!empty($this->wrongMethodNames)) {
```

Comment:
`empty` is redundant here, empty arrays are always `false` and if it stops being an array I'd prefer a hard break